src/tests/Microsoft.PowerFx.Core.Tests/PowerFxTest.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Globalization;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public class PowerFxTest
+    {
+        static PowerFxTest()
+        {
+            // Force all tests to use en-US locale
+            PowerFxConfig.CultureOverride = new CultureInfo("en-US");
```

Comment:
>PowerFxConfig.CultureOverride [](http://example.com/codeflow?start=12&length=29)

Could we avoid introducing a new static PowerFxConfig.CultureOverride , and just set the existing static, CultureInfo.CurrentCulture

This maight let us avoid any changes to product FxCore, and just changes to test.